recipe/bld.bat

```diff
@@ -1,8 +1,12 @@
 mkdir build && cd build
 
-cmake .. -G "NMake Makefiles"                  ^
-	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%    ^
-	-DCMAKE_BUILD_TYPE=Release                 ^
-	-DBUILD_SHARED_LIBS=OFF
+cmake .. -G "%CMAKE_GENERATOR%" ^
+	-DBUILD_SHARED_LIBS=ON ^
+	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+	-DENABLE_ZLIB_COMPRESSION=ON ^
+	-DCMAKE_BUILD_TYPE=Release ^
+	-DENABLE_DEBUG_LOGGING=ON
```

Comment:
> Should I turn it off then?

In my limited experience debug mode is usually slower and only needed for special cases. So unless you have that special case I would recommend to remove it.

PS: and if you do have an special case we should probably create another package like `libssh2-debug` for example. But there is no precedente nor rule for that in conda-forge yet. (We should probably discuss that though.)